.circleci/bin/test-airflow-image.py

```diff
@@ -85,7 +90,21 @@ def test_redis_version(webserver):
         raise Exception("redis pip module is not installed")
     version = redis_module['version']
     assert semantic_version(version) != semantic_version('3.4.0'), \
-           "redis module must not be 3.4.0"
+        "redis module must not be 3.4.0"
+
+
+def test_astronomer_airflow_check_version(webserver):
+    """ astronomer-airflow-version-check 1.0.0 has an issue in the Astronomer platform
+    """
+    try:
+        version_check_module = webserver.pip_package.get_packages()['astronomer-airflow-version-check']
+    except KeyError:
+        print("astronomer-airflow-version-check pip module is not installed")
+        return
+    version = version_check_module['version']
+    assert semantic_version(version) > semantic_version('1.0.0'), \
```

Comment:
```suggestion
    assert semantic_version(version) >= semantic_version('1.0.1'), \
```

Captures the intent better I feel.